src/test/compile-fail/cast-rfc0401.rs

```diff
@@ -59,10 +59,12 @@ fn main()
     //~^ ERROR casting
     //~^^ HELP through a usize first
     let _ = 3_i32 as bool;
-    //~^ ERROR cannot cast as `bool`
+    //~^ ERROR cannot cast as `bool` [E0054]
```

Comment:
I'm surprised that putting the `[E0054]` at the end passes the tests.  For example, when I run this test by hand I get this as output:

```
error[E0054]: cannot cast as `bool`
  --> src/test/compile-fail/cast-rfc0401.rs:61:13
   |
61 |     let _ = 3_i32 as bool;
   |             ^^^^^^^^^^^^^
   |
   = help: compare with zero instead
```

Notice the `[E0054]` part is on the left and not the right.  You may need to update your compiler source.